### PR TITLE
Create 2022-12-16-continuous-request-sizing.md

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

**Comment by bstuder99** [comment]
Image links have been double-checked and should display properly.

File: _posts/2022-12-16-continuous-request-sizing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tags:
+  - right-sizing
+  - application
+  - workload
+  - saving
+---
+
+Ensuring your application requests are efficient is key to saving on your Kubernetes infrastructure. If Vertical Pod Autoscaling is not the best fit for your workloads, or you want to avoid a tedious setup, Kubecost can provide you with a simpler solution: continuous request right-sizing, available today.
+
+Kubecost has provided basic right-sizing recommendations for cluster nodes and PVs, but goes one step further by now being able to analyze your workload requests. This dynamic approach to optimizing your workloads begins by monitoring your workloads and provides recommendations for optimizing your requests. By checking the gaps between requests and usage, we find how to shorten that gap, saving you money by ensuring your resources are used more efficiently. These recommendations can be implemented with just one click, and they can be applied on a recurring basis. You can customize the frequency to apply these recommendations by a custom interval, including minutes, days, or weeks to best suit your needs.
```
Comment (michaelmdresser) [inline _posts/2022-12-16-continuous-request-sizing.md:18]: > now being able to analyze your workload requests

We've had this functionality for a long time, might be worth a rephrase. Perhaps "has provided ... recommendations for requests, nodes, and PVs. Now, Kubecost goes one step further by giving you the ability to continuously implement request recommendations. This dynamic approach ..."

**Comment by michaelmdresser** [comment]
Hey @bstuder99 I think this merged with a partially complete sentence from my suggestion. Maybe some wires crossed?

**Comment by bstuder99** [comment]
Yes I saw it and fixed it immediately, fix should have deployed already

**Comment by michaelmdresser** [comment]
Oh excellent, thanks.

**Pull request merged**